node/src/triple.rs

```diff
@@ -36,7 +36,13 @@ pub async fn run_many_triple_generation<const N: usize>(
         me.into(),
         threshold,
     )?;
+
+    let start = SystemTime::now();
+
     let triples = run_protocol("many triple gen", channel, me, protocol).await?;
+
+    let duration = start.elapsed()?.as_millis();
+    metrics::MPC_NUM_TRIPLES_GENERATION_TIME_MS.set(duration as i64);
```

Comment:
I would propose to rename the variable to `MPC_LAST_NUM_TRIPLES_GENERATION_TIME_MS`, to better reflect what it tracks.

Given that below we track the total number of triples generated, it could be beneficial to additionally track the total time spent generating triples